src/managers/ReactionManager.js

```diff
@@ -1,22 +1,30 @@
 'use strict';
 
-const DataStore = require('./DataStore');
 const MessageReaction = require('../structures/MessageReaction');
+const BaseManager = require('./BaseManager');
 
-/**
- * Stores reactions.
- * @extends {DataStore}
- */
-class ReactionStore extends DataStore {
+class ReactionStore extends BaseManager {
```

Comment:
```suggestion
class ReactionManager extends BaseManager {
```

Also change the export